extensions/wikia/BannerNotifications/BannerNotificationsController.class.php

```diff
@@ -5,7 +5,7 @@
  * @author Maciej Brencz
  */
 
-class NotificationsController extends WikiaController {
+class BannerNotificationsController extends WikiaController {
```

Comment:
We'll have to warn QA that the oasis notifications will need a pass since they are affected.